tflint/config.go

```diff
@@ -477,6 +522,11 @@ func (c *Config) enableBundledPlugin() *Config {
 	return c
 }
 
+// IsJSONConfig returns true if the configuration was loaded from a JSON file
+func (c *Config) IsJSONConfig() bool {
+	return c.isJSONConfig
```

Comment:
Alternatively, you could just store the config path on the config object, unexported. Or expect it as the first/only key in `sources`. And then call `return filepath.Ext(filename) == ".json"` here